flow/activities/flowable.go

```diff
@@ -767,6 +768,19 @@ func (a *FlowableActivity) RecordSlotSizes(ctx context.Context) error {
 					return
 				}
 				slotMetricGauges.IntervalSinceLastNormalizeGauge = intervalSinceLastNormalizeGauge
+
+				syncedTablesGauge, err := a.OtelManager.GetOrInitInt64Gauge(
+					otel_metrics.BuildMetricName(otel_metrics.SyncedTablesGaugeName),
```

Comment:
either now or after we should make `RecordGauge` utility method on otelmanager

Reply:
Yeah, I feel that all gauges/counters can probably be initialized in otel manager. There is no flow specific information when we initialize them